inc/views/inpost/seo-settings.php

```diff
@@ -134,135 +134,162 @@
 
 		?>
 		<?php if ( 'above' === $this->inpost_seo_bar || $this->is_option_checked( 'display_seo_bar_metabox' ) ) : ?>
-		<p>
-			<strong><?php esc_html_e( 'Doing it Right', 'autodescription' ); ?></strong>
-			<div>
-				<?php $this->post_status( $post_id, 'inpost', true ); ?>
+		<div class="tsf-singular-metabox">
+			<div class="tsf-singular-metabox-label">
+				<strong><?php esc_html_e( 'Doing it Right', 'autodescription' ); ?></strong>
 			</div>
-		</p>
+			<div class="tsf-singular-metabox-input">
+				<div>
+					<?php $this->post_status( $post_id, 'inpost', true ); ?>
+				</div>
+			</div>
+		</div>
 		<?php endif; ?>
 
-		<p>
-			<label for="autodescription_title"><strong><?php printf( esc_html__( 'Custom %s Title', 'autodescription' ), esc_html( $type ) ); ?></strong>
-				<a href="<?php echo esc_url( 'https://support.google.com/webmasters/answer/35624?hl=' . $language . '#3' ); ?>" target="_blank" title="<?php esc_attr_e( 'Recommended Length: 50 to 55 characters', 'autodescription' ); ?>">[?]</a>
-				<span class="description tsf-counter">
-					<?php printf( esc_html__( 'Characters Used: %s', 'autodescription' ), '<span id="autodescription_title_chars">' . (int) mb_strlen( $tit_len_parsed ) . '</span>' ); ?>
-					<span class="hide-if-no-js tsf-ajax"></span>
-				</span>
-			</label>
-		</p>
-		<p>
-			<div id="tsf-title-wrap">
-				<input class="large-text" type="text" name="autodescription[_genesis_title]" id="autodescription_title" placeholder="<?php echo esc_attr( $doctitle_placeholder ); ?>" value="<?php echo esc_attr( $this->get_custom_field( '_genesis_title' ) ); ?>" />
-				<span id="tsf-title-offset" class="hide-if-no-js"></span><span id="tsf-title-placeholder" class="hide-if-no-js"></span>
+		<div class="tsf-singular-metabox">
+			<div class="tsf-singular-metabox-label">
+				<label for="autodescription_title"><strong><?php printf( esc_html__( 'Custom %s Title', 'autodescription' ), esc_html( $type ) ); ?></strong>
+					<a href="<?php echo esc_url( 'https://support.google.com/webmasters/answer/35624?hl=' . $language . '#3' ); ?>" target="_blank" title="<?php esc_attr_e( 'Recommended Length: 50 to 55 characters', 'autodescription' ); ?>">[?]</a>
+					<span class="description tsf-counter">
+						<?php printf( esc_html__( 'Characters Used: %s', 'autodescription' ), '<span id="autodescription_title_chars">' . (int) mb_strlen( $tit_len_parsed ) . '</span>' ); ?>
+						<span class="hide-if-no-js tsf-ajax"></span>
+					</span>
```

Comment:
Move this span block just below the label tag, it's weird now when clicking on it (the label on-click affects focusing on the input container).